ExtraTasks-1.playground/Contents.swift

```diff
@@ -0,0 +1,95 @@
+//1. what digit is bigger ?
+
+let height = 15
+let width  = 20
+
+if height > width {
+    print("Height value: \(height). Height is bigger")
+} else if height < width {
+    print("Width value: \(width). Width is bigger")
+} else {
+    print("The values are the same")
+}
+
+
+/*2. Check if we take all that we need to go for a work
+Нам надо идти на работу, но нужно проверить все ли мы взяли или нет?
+Нам точно нужны ключи документы и ручка, но из еды нам надо яблоко или апельсин.
+Для решения этой задачи нам помогут логические операторы **`|| &&`**.
+*/
+
+let hasKey       = true
+let hasDocuments = true
+let hasPen       = true
+let takenApple   = false
+let takenOrange  = true
+var shouldGoToWork : Bool
```

Comment:
You can improve this line by putting the bool value here right away and using it as a condition after `if`. You're just doing extra work assigning value to it inside `if else`